src/components/decklist/decklist.tsx

```diff
@@ -184,3 +192,18 @@ function getColumnMode(viewMode: ViewMode, group: DeckGrouping) {
   if (viewMode === "scans") return "scans";
   return countGroupRows(group) < 5 ? "single" : "auto";
 }
+
+function computeXPSum(grouping: DeckGrouping | undefined) {
+  if (!grouping) return 0;
+
+  const xpGroups = resolveXP(grouping);
+
+  return xpGroups.entries().reduce((acc, [code, xp]) => {
+    // This looks hacky is there a better way ?
```

Comment:
This looks hacky 🤔 Is there a better way ?

Reply:
Yes, given that we don't _need_ to act on a grouping level here, we can use a simpler method to compute this. I think it could be something like (pseudo code):

```ts
function computeXPSum(deck: ResolvedDeck, slotKey: "sideSlots" | "slots") {
  if (!deck[slotKey]) return 0;

  const xp = Object.entries(deck[slotKey]).reduce((acc, [code, quantity]) => {
    const card = deck.cards[slotKey][code]?.card;
    return card ? acc + countExperience(card, quantity) : acc;
  }, 0);

  return xp;
}
```